lpa-submissions-e2e-tests/cypress/integration/accessNeighboursLand/accessNeighboursLand.step.js

```diff
@@ -2,10 +2,10 @@ import { Before } from 'cypress-cucumber-preprocessor/steps';
 
 const page = {
   id: 'accessNeighboursLand',
-  heading: "Would the inspector need access to a neighbour's land?",
+  heading: "Would the Inspector need access to a neighbour's land?",
   section: 'About the Appeal Site',
   title:
-    "Would the inspector need access to a neighbour's land? - Appeal questionnaire - Appeal a householder planning decision - GOV.UK",
+    "Would the Inspector need access to a neighbour's land? - Appeal questionnaire - Appeal a householder planning decision - GOV.UK",
   url: 'neighbours-land',
   emptyError: 'Select yes if the inspector needs access to a neighbour’s land',
```

Comment:
Check with @pippalou if we need to change 'inspector' to 'Inspector'?

Reply:
@shilpabasu22 yes its needs to be Inspector, however in dev the error message already has a capital I?

Reply:
Thanks both. I have updated the content.